tests/pages/subjects/SubjectDetailsPage.test.js

```diff
@@ -0,0 +1,81 @@
+import { mount, shallow } from 'enzyme';
```

Comment:
Test passes with 3 warnings:
- Warning: Failed prop type: Invalid prop `subjectId` of type `number` supplied to `SubjectDetailsPage`, expected `string`.
- Warning: Failed prop type: The prop `children` is marked as required in `Brevier`, but its value is `undefined`.
- Warning: Failed prop type: The prop `children` is marked as required in `DynamicTextComponent`, but its value is `undefined`.